src/Parser/HavocDemonHunter/Modules/Statistics/Spells/Nemesis.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+import Icon from 'common/Icon';
+
+import { formatPercentage } from 'common/format';
+import { formatDuration } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class Nemesis extends Module {
+
+  statistic() {
+
+    const nemesisUptime = this.owner.modules.enemies.getBuffUptime(SPELLS.NEMESIS_TALENT.id);
+    const nemesisUptimePercentage = nemesisUptime / this.owner.fightDuration;
+
+    if(nemesisUptime > 0)
```

Comment:
Please don't make statistics optional like this; a consistent layout is important and so is giving positive feedback instead of leaving the user to wonder why it isn't showing.

If this depends on some sort of talent or trait it should be checked in the initialized event.

Reply:
Gonna do it on the next PR.